src/odemis/gui/cont/tabs.py

```diff
@@ -665,6 +655,19 @@ def get_display_priority(cls, main_data):
         else:
             return None
 
+    @call_in_wx_main
+    def display_acquired_data(self, data):
+        """
+        Displays the acquired streams on the top right view
+        data (DataArray): the images/data acquired 
+        """
+        # get the top right view port
+        view = self.tab_data_model.views.value[1]
+        for s in data_to_static_streams(data):
+            stream_cont = StreamController(self.panel.pnl_cryosecom_acquired, s, self.tab_data_model,
+                                           show_panel=True, view=view, sb_ctrl=self._overview_stream_controller)
+            stream_cont.stream_panel.collapse(True)
```

Comment:
This is confusing. It might be correct what you are saying, but Eric told me to only add the acquired stream to the acquired streams panel as you will be working here further. So either you take care of this, or tell me the big picture as well as details of what you want to do here, because this part is all not mentioned in the specifications.

Reply:
Sorry for badly communicating it. @lazem , my idea is that Mahmoud just puts "rough" code, and you'll later put all the proper code. It's too much fluctuating right now.
@Mahmood-B , could you just add a `# TODO adjust this code to fit the Feature behaviour` at the top.